source/nodejs/adaptivecards/src/carousel.ts

```diff
@@ -605,6 +605,17 @@ export class Carousel extends Container {
             this.raiseCarouselEvent(Enums.CarouselInteractionEvent.Autoplay);
         });
 
+        carousel.on('paginationRender', (swiper, paginationEl) => {
```

Comment:
nit/question: Did we want to enable the keyboard arrow keys on the carousel container?

Reply:
arrow key is enabled, is just that pressing arrow key won't turn carousel pages.

Reply:
@jwoo-msft yeah I meant that if we implemented turning the carousel pages ourselves, would it work as expected (not turn all widgets)?

I like the feature in general, so I was just curious :)

Reply:
@anna-dingler, we probably run into the same issue they are facing of figuring out where the event occurred.